lambda/shared/src/main/scala/feral/lambda/LambdaEnv.scala

```diff
@@ -29,7 +29,7 @@ import cats.kernel.Monoid
 import cats.syntax.all._
 import cats.~>
 
-sealed trait LambdaEnv[F[_], Event] { outer =>
+trait LambdaEnv[F[_], Event] { outer =>
```

Comment:
I feel like a more appropriate name for this is `LambdaInvocation` but ... that's a mouthful, and it's a bit late in the day for changes :)